src/service/persons.with.signficant.control.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { Request } from "express";
+import { makeApiCallWithRetry } from "./retry.handler.service";
+import { logger } from "../utils/logger";
+import { CompanyPersonsWithSignificantControlResource } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+
+export const getCompanyPsc = async (
+  req: Request,
+  companyNumber: string,
+): Promise<CompanyPersonsWithSignificantControlResource> => {
+  const response = await makeApiCallWithRetry(
+    "companyPsc",
+    "getCompanyPsc",
+    req,
+    req.session as Session,
+    companyNumber
+  );
+
+  logger.debugRequest(req, `Received company PSC data ${JSON.stringify(response)}`);
```

Comment:
No need to show full json on debug. Could do that on Trace but you should be able to turn on debug without seeing lengthy JSON responses.

Reply:
company profile just shows the OE number.

Reply:
updated to just show simple message that PSC data retrieved for given OE number